src/components/presentational/Chat/Chat.css

```diff
@@ -0,0 +1,75 @@
+.mdhui-chat {
+    background: var(--mdhui-background-color-0);
+    font-size: 14px;
+}
+
+.mdhui-chat .mdhui-chat-log {
+    overflow-y: auto;
+    overflow-x: hidden;
+    min-height: 20vh;
+}
+
+.mdhui-chat p,
```

Comment:
I would remove these styles unless there's a strong need to override them; e.g. just go with the default styles

Reply:
Removed the text color as that wasn't really needed, the other 2 do change behavior? Less top and bottom margin and different word break behavior than default (normal)?